lib/app/pages/add_folder_words_page/add_folder_words_page.dart

```diff
@@ -147,13 +152,29 @@ class _BodyState extends State<_Body> {
             child: widget.wordsList.isNotEmpty ? AddedWordsGrid(wordsList: widget.wordsList) : null,
           ),
           CustomTextField(
+            prefix: Padding(
+              padding: const EdgeInsets.only(top: AppDimensions.d8, right: AppDimensions.d8),
+              child: CountryFlag.fromLanguageCode(
+                'de',
+                height: AppDimensions.d26,
+                width: AppDimensions.d26,
+              ), //TODO:Add flag
+            ),
             error: _initialWordError,
             controller: _initialWordController,
             hintText: context.tr.translationWordDesc,
             onChanged: _onChangedInitialWord,
           ).animate().slideX().fade(),
           if (_isInitialWordNotEmpty)
             CustomTextField(
+              prefix: Padding(
+                padding: const EdgeInsets.only(top: AppDimensions.d10, right: AppDimensions.d8),
+                child: CountryFlag.fromLanguageCode(
+                  'en',
+                  height: AppDimensions.d26,
+                  width: AppDimensions.d26,
+                ), //entity.language
```

Comment:
always remember to add a TODO to the comment, it makes it easier to find it later